config/notifiers.go

```diff
@@ -449,6 +450,15 @@ func (c *VictorOpsConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	if c.RoutingKey == "" {
 		return fmt.Errorf("missing Routing key in VictorOps config")
 	}
+
+	reservedFields := []string{"routing_key", "message_type", "state_message", "entity_display_name", "monitoring_tool", "entity_id", "entity_state"}
```

Comment:
Users should not be able to override the routing_key, as that's the group key

Reply:
Since routing key is placed in the URL, it still felt awkward to allow the user to define another field as routing key in the custom fields. Thoughts?

Reply:
@jrthrawny I've got no strong feeling on this. It doesn't create a conflict per-se but as you said, it probably doesn't make sense to send it as a custom field.

Reply:
Any further feedback here? I still think allowing the user to add routing_key as a custom field could be confusing and misleading, therefor not allowed. Certainly willing to consider otherwise.

Reply:
@jrthrawny I agree with you.